socorro/unittest/external/postgresql/unittestbase.py

```diff
@@ -19,11 +21,15 @@ class PostgreSQLTestCase(TestCase):
     required_config = Namespace()
     required_config.namespace('database')
     required_config.add_option(
+        name='database_url',
+        default='postgres://test:aPassword@localhost:5432/socorro_integration_test',
```

Comment:
If the default was `'postgres://localhost:5432/socorro_integration_test'` instead this'd work for most people out-of-the-box because then internally it will connect with the user running nose and with a blank password which is a common setup for local dev.